.github/workflows/build.yml

```
@@ -38,3 +53,8 @@ jobs:
         env:
           SHARED: ${{ matrix.shared }}
         run: make
+      - name: Generate artifacts (shared)
+        uses: actions/upload-artifact@v3
+        with:
+          name: hashcat-${{ matrix.os }}-${{ matrix.shared }}
+          path: ${{ env.include_paths }}
```

Instead of names like `hashcat-macos-latest-0` or `hashcat-ubuntu-latest-1` use more appropriate names like `Hashcat-Linux-Static` or `Hashcat-MacOS-Shared`.

```suggestion
      - name: Generate static artifacts
        if: ${{ matrix.shared == 0 }}
        uses: actions/upload-artifact@v3
        with:
          name: Hashcat-${{ runner.os }}-Static
          path: ${{ env.include_paths }}
      - name: Generate shared artifacts
        if: ${{ matrix.shared == 1 }}
        uses: actions/upload-artifact@v3
        with:
          name: Hashcat-${{ runner.os }}-Shared
          path: ${{ env.include_paths }}
```